src/vs/platform/tunnel/common/tunnel.ts

```diff
@@ -416,7 +416,7 @@ export abstract class AbstractTunnelService implements ITunnelService {
 		const hostMap = this._tunnels.get(remoteHost);
 		if (hostMap) {
 			const tunnel = hostMap.get(remotePort);
-			const tunnelResult = await tunnel;
+			const tunnelResult = await tunnel.value;
```

Comment:
It's not clear to me if we can assume `tunnel` is non-null here, or if we need to add a check.